frontend/tailwind.config.js

```diff
@@ -196,16 +277,20 @@ module.exports = {
 		},
 		margin: (theme, { negative }) => ({
 			auto: 'auto',
+			'screen': 'calc(50% - 50vw)',
 			...theme('spacing'),
 			...negative(theme('spacing'))
 		}),
 		maxHeight: {
 			full: '100%',
-			screen: '100vh'
+			screen: '100vh',
+			'37': '37rem',
```

Comment:
Also we should remove this project-specific styles.

Reply:
I'll remove them